src/day-1-toy/comp.py

```diff
@@ -1,14 +1,15 @@
 # Write a list comprehension to produce the array [1, 2, 3, 4, 5]
 
-y = []
+y = [i for i in range(1, 6)]
 
 print (y)
 
 # Write a list comprehension to produce the cubes of the numbers 0-9:
 # [0, 1, 8, 27, 64, 125, 216, 343, 512, 729]
 
 y = []
-
+for x in range(10):
+    y.append(x**3)
 print(y)
```

Comment:
How would you do the above in a list comprehension? Remember:
```
y = [<return value/transformed element> for <element> in <list/iterable>]
```

Reply:
you'll see these updated now!